src/utils/types.ts

```diff
@@ -0,0 +1,50 @@
+type Image = {
+  path: string;
+  alt: string;
+  type?: string;
+};
+
+type Programme = {
+  name: string;
+  link: string;
+  icon: string;
+};
+
+type WCCEvent = {
```

Comment:
what do you think about `CommunityEvent` or just `Event`?

Reply:
updated - went with `Event` now 😅